c++/Makefile.am

```diff
@@ -482,6 +482,9 @@ capnp_test_LDADD = libcapnp-test.a libcapnp.la libkj-test.la libkj.la
 else !LITE_MODE
 
 check_PROGRAMS = capnp-test capnp-evolution-test capnp-afl-testcase
+if !HAS_NO_FUZZING_ENGINE
```

Comment:
Sorry, this is nitpicky but I think it's the last thing... This is a double-negative. I think it would be easier to read as a positive, i.e. the variable should be `HAS_FUZZING_ENGINE`.

Reply:
Thought the same
No problem with being nitpicky
Looks like this double-negative build succeeded in building the fuzz target